src/it/java/org/jboss/tools/intellij/quarkus/utils/HelperUtils.java

```diff
@@ -23,6 +23,10 @@
 public class HelperUtils {
 
     public static String listOfRemoteTextToString(List<RemoteText> data) {
+        if (data.isEmpty()) {
```

Comment:
Why is this required ?

Reply:
Because I need to return a not null String from that method. I usually call some method to search inside the String etc. I see it more simple to check it inside the method than the every time outside the method. But if you see it more clearly implemented not to check it here, I will rewrite the tests.

Reply:
If you remove that piece of code but execute the result you will get an empty string as result,no ?

Reply:
Oh yes, you are right. I did understand the code wrong, after I have executed it now I finally see that it is not required -> removed.